scripts/phase2b_client_control.py

```
@@ -0,0 +1,562 @@
+"""
+Phase 2b: Client Remote-Control Protocol & Query Parameters
+"""
+
+import yaml
+from copy import deepcopy
+
+SPEC_PATH = "plex-api-spec.yaml"
+
+
+def load_spec():
+    with open(SPEC_PATH, "r") as f:
```

Ruff (0.15.15)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)